lib/safemode/core_jails.rb

```diff
@@ -239,12 +239,12 @@ def core_jail_class_methods(klass)
     abs
     blank?
     ceil
-    chr
     coerce
     div
     divmod
     downto
     floor
+    chr
```

Comment:
which locale did you use for sorting? :-) feels like czech `ch`

Reply:
Yes, I can confirm. Realized this too late, I am going to keep it in this Czech order and before squash I can fix this. Not big deal other than our `ch` character.